lib/libpreopen.c

```diff
@@ -38,7 +37,10 @@
 #include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
-
+#include <sys/mman.h>
```

Comment:
Again, kernel include files should come first (and be in alphabetical order, as far as possible).